.solcover.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    norpc: true,
+    compileCommand: '../node_modules/.bin/truffle compile',
+    testCommand: '../node_modules/.bin/truffle test --network coverage',
+    copyPackages: [],
+    skipFiles: [
+        'test/TestImports.sol',
+    ]
+}
```

Comment:
We should use a similar one to our own apps, e.g. https://github.com/aragon/aragon-apps/blob/master/apps/voting/.solcover.js

Reply:
I could do a pull request to the `aragon-apps` repo. Do you want me to do it?

Reply:
Actually I meant pulling over the one from `aragon-apps` to this repo 😄

Reply:
Ok, done!